src/main/java/edu/oakland/production/display01/DisplayGpsReceiverImplementation.java

```diff
@@ -24,9 +39,9 @@ public String measureGpsSignalStrength(Satellite satelliteSignal) {
    * @return Request to reconnect to satellite.
    */
   public SatelliteSignalCheckRequest measureSignal(Satellite satelliteSignal) {
-        
-    return null;
-
+    if (satelliteSignal == null) {
+      throw new IllegalArgumentException("Signal strength cannot be null");
```

Comment:
Just to make this exception a little more descriptive, can we change the output to the name of the parameter? Thank you!
```suggestion
      throw new IllegalArgumentException("satelliteSignal cannot be null");
```